### PR TITLE
Use Akka Stream TestKit in the tests

### PR DESCRIPTION
**Pull request by cchantep**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -99,18 +99,26 @@ lazy val `anorm-iteratee` = (project in file("iteratee"))
...
+
 lazy val `anorm-akka` = (project in file("akka"))
   .enablePlugins(PlayLibrary)
   .settings(scalariformSettings: _*)
   .settings(
     previousArtifacts := Set.empty,
+    resolvers ++= Seq(
+      // For Akka Stream TestKit 'tests' (see akka/akka#21028)
+      "Tatami Releases".at(
+        "https://raw.github.com/cchantep/tatami/master/releases")),
```
Comment (schmitch) [inline build.sbt]: what's that? It gives me a 404

Comment (cchantep) [inline build.sbt]: That's the repository base URL, directory listing is not enabled (and not required for the dependency resolution); e.g. https://raw.github.com/cchantep/tatami/master/releases/com/typesafe/akka/akka-stream-testkit_2.11/2.4.8/akka-stream-testkit_2.11-2.4.8.pom.md5

**Comment by schmitch** [comment]
LGTM

**Comment by cchantep** [comment]
Could this one be merged now?

**Pull request merged**